tests/__init__.py

```diff
@@ -114,10 +114,11 @@ def clear_views(cursor):
         )
 
 
-def truncate_tables(cursor):
-    cursor.execute("TRUNCATE TABLE files CASCADE")
-    cursor.execute("TRUNCATE TABLE bundles CASCADE")
-    cursor.execute("TRUNCATE TABLE bundles_files CASCADE")
-    cursor.execute("TRUNCATE TABLE job_status CASCADE")
-    cursor.execute("TRUNCATE TABLE process_links_join_table CASCADE")
-    cursor.execute("TRUNCATE TABLE hca_processes CASCADE")
+def truncate_tables():
+    # config.db_session.execute("TRUNCATE TABLE files CASCADE")
+    # config.db_session.execute("TRUNCATE TABLE bundles CASCADE")
+    # config.db_session.execute("TRUNCATE TABLE bundles_files CASCADE")
```

Comment:
Is there a reason for these lines to be commented out? If there is a long-lasting reason, please put it in a comment line above them.

Is this related to our discussion about not blowing away the database at the beginning of every test?

Reply:
removed